extension/sqlsmith/fuzzyduck.cpp

```diff
@@ -39,7 +39,7 @@ void FuzzyDuck::Fuzz() {
 	BeginFuzzing();
 	for (idx_t i = 0; i < max_queries; i++) {
 		LogMessage("Query " + to_string(i) + "\n");
-		auto query = GenerateQuery();
+		auto query = GenerateQuery(GenerateRandomNuber() % 10);
```

Comment:
Can we add a percentage check or something similar? I.e 90% of the time we can just test single statements.
10% of the time it should be a multi statement and then we can generate a random number for how many statements we will generate

Reply:
Yes, will add it to the GenerateQuery()